clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -116,12 +114,21 @@ private void maybeAutoCommit(final long currentTimeMs) {
         log.debug("Auto-committing offsets {}", allConsumedOffsets);
```

Comment:
I feel this message is misleading - but in the new model, we batch the commits and send them out at the network client poll in the background thread. Maybe we word it as "enqueuing autocommit offsets {}", offsets ?